DIPs/DIP-25.md

```diff
@@ -0,0 +1,49 @@
+---
+DIP: 25
+Title: Community Hub - Temporary Anonymous Zone
+Status: Accepted
+Themes: Community Involvement
+Tags: Event Production
+Authors: Althea <althea@ethereum.org>
+Resources Required: Physical space, signage, seating, projector if possible
```

Comment:
Projector is not possible, all hubs will be equipped with one TV-sized screen though (see Community Hub production guide)

Reply:
Oh sorry, knew that but forgot to update. Thank you!